### ISSUE TITLE
Accessibility: MAS40B: Inspect : Name property is empty for the "Trending" list Box

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#50]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/VSCode in EDGE
2. Navigate to the "Trending" list box available under the "Most Popular" extensions 
3. Observe the "Name" property in Inspect

**Actual Result:**
The "Name" property is empty

**Narrator Behaviour:**
The Narrator announces as "Group"

**Expected Result:**
The "Name" property should be specific
Eg: "Trending List Box"

**User Impact:**
Visually impaired users  will not get to know the purpose of the list box as the screen reader will announce as "group" when the user arrives at the control. Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilities.

**Screenshots:**
![name property is empty for trending list box](https://cloud.githubusercontent.com/assets/24825441/21718303/9fae86f0-d43c-11e6-9320-af3e89783462.png)

**Comment by waderyan** [comment]
Marketplace

**Comment by waderyan** [comment]
Please send to Marketplace team.
